airbyte-integrations/connectors/source-iterable/source_iterable/api.py

```diff
@@ -272,7 +161,7 @@ class EmailOpen(IterableExportStream):
     data_field = "emailOpen"
 
 
-class EmailSend(IterableExportStream):
+class EmailSend(IterableExportStreamRanged):
```

Comment:
Can't we apply the same method for other `email` and `template` streams as well, it will be more reliable?

Reply:
No, it would generate a lot of requests and reach limit of 4 requests per minute.  Using it only for email_send stream as this stream caused fails on user side.

Reply:
It will not be very logical if, knowing how to handle the rest of the streams, we will wait until they break. EmailClick, EmailOpen and EmailSendSkip are pretty heavy streams too.

Reply:
Done